Automattic/wp-calypso
# A4A > Agency Tier: Implement celebration modal responsiveness

Closes https://github.com/Automattic/automattic-for-agencies-dev/issues/1312

## Proposed Changes

This PR implements:

- Mobile view for the celebration modal.
- Overflowing content indicator for the celebration modal.

## Why are these changes being made?

* To enhance the mobile view and show the overflowing content indicator for the celebration modal.

## Testing Instructions

* Update your agency tier to **Emerging Partner** using the MC tool > Open the A4A live link and verify that the screen looks as shown below. You can see a faded horizontal div when the content is overflowing, which disappears when scrolled to the bottom.

https://github.com/user-attachments/assets/20eb1a0c-201f-45a5-8270-75c4fd4e3721

* Switch to mobile and verify the screen looks as shown below and you can scroll to see the overflowing content.

![iPhone-12-PRO-agencies localhost (5)](https://github.com/user-attachments/assets/4a6c3b08-2140-420d-a13f-a979d357ae90)

* Switch the desktop view > Change the agency tier to **Agency Partner** > Refresh the UI and verify the screen looks good.

* Switch to mobile and verify the screen looks as shown below and you can scroll to see the overflowing content.

![iPhone-12-PRO-agencies localhost (7)](https://github.com/user-attachments/assets/1f2c370e-93c9-4c61-8e7b-b06d56f3d97c)

* Change the agency tier to **Pro Agency Partner** > Refresh the UI and verify the screen looks good.

* Switch to mobile and verify the screen looks as shown below and you can scroll to see the overflowing content.

![iPhone-12-PRO-agencies localhost (8)](https://github.com/user-attachments/assets/fe4affe9-3dd4-4885-b916-f2bfdc5f8d2e)

## Pre-merge Checklist
